chat-commands.js

```diff
@@ -804,6 +810,7 @@ exports.commands = {
 				return this.errorReply(`This room is already public.`);
 			}
 			if (room.isPersonal) return this.errorReply(`This room can't be made public.`);
+			if (room.parent) return this.errorReply(`Subrooms cannot be public.`);
```

Comment:
Tour games are often public, and they're subrooms of the tour room.

Reply:
You haven't dealt with this yet...

Reply:
Done.